distributions/openhab/src/main/resources/bin/update

```diff
@@ -62,7 +62,7 @@ setup(){
   fi
 
   ## Check to see if processes are running before updating
-  if [ ! -z "$(ps aux  | grep openhab | grep -v grep)" ]; then
+  if [ ! -z "$(ps aux  | grep "openhab.*java" | grep -v grep)" ]; then
```

Comment:
```sh
if [ ! -z "$(pgrep -f "openhab.*java")" ]; then
```

Would be cleaner, and [maybe safer](https://github.com/openhab/openhab-linuxpkg/pull/146) to use if this works correctly on MacOS?

Reply:
Just tested, `pgrep` seems to work on macOS as well, yes. Wanna do a PR?